dist/tools/renode/README.md

```diff
@@ -0,0 +1,25 @@
+# Emulation using Renode
+
+## Introduction
+[Renode](http://renode.io) is a virtual development tool for multinode embedded networks (both wired and wireless) enabling a scalable workflow for building effective, tested and secure IoT systems, created by [Antmicro](http://antmicro.com/blog/2017/08/renode-press-release/).
+It can easily be used to run applications on a broad range of embedded platforms without any changes in the code itself, as if you were running on real hardware - but with more possibilities.
+
+## Installation
+
+### From package
+Packages for macOS, deb-based and rpm-based systems, for Windows and for Arch Linux are available on [GitHub](https://github.com/renode/renode/releases/latest).
+
+### From source
+Follow the installation instructions on Renode's [GitHub](https://github.com/renode/renode#installation) page.
+
+After compilation is successful, ensure that `renode` is available on your `PATH`. One way to do so, is via symlink: `sudo ln -s path/to/renode/repository/run.sh /usr/local/bin/renode`.
+
+### Testing
+After installation, verify if Renode is working using `renode --help`. You should be presented with a help screen.
+
+## Usage
+From within RIOT-OS, you can use `make emulate` to start emulation. It expects a board definition file in `boards/<BOARD>/dist/board.resc`.
```

Comment:
Is there any doc on how to write such a `board.resc` file? If yes, it might be a good idea to link it here.

Reply:
According to https://github.com/renode/renode/, documentation is work in progress :-)

However, some work in progress documentation can be found here: https://renode.readthedocs.io/en/latest/.

Reply:
Even a link to a WIP documentation would be appreciated ;-).